### DISCUSSION TITLE
Commit Names as Dependencies

### DISCUSSION THREAD

**Discussion by DasLixou** [renovatebot/renovate#15623] [Request Help]
Hello!
Renovate only updates the dependencies of gradle that use semantic versioning. Is there a way to configure a github repo next to a dependency so Renovate can also update dependencies with versions from JitPack like `1dc817b`?

**Comment by viceice** [comment]
renovate can't order versions which only consist of a hash. so it don't know which one is an update 

please provide some more details, so we can maybe find a workaround.

**Reply by DasLixou** [reply]
So I use https://github.com/Minestom/Minestom and they use Jitpack so every commit they make, Jitpack will automatically make a new artifact on their repository with the version name of the commit it’s made of. And I want that renovate updates Minestom too if that’s possible. Maybe renovate has a feature to set a GitHub repo where it can look if there are new commits or so. That was my question
